docs/docs/babel.md

```diff
@@ -2,19 +2,20 @@
 title: Babel
 ---
 
-Gatsby uses the phenomenal project [Babel](https://babeljs.io/) to enable support for writing modern JavaScript — while still supporting older browsers.
 
-## How to specify which browsers to support
+जुन्या ब्राऊजरला समर्थन देत असताना- गॅटस्बी [babel](https://babeljs.io/) आधुनिक जावास्क्रिप्ट लिहिण्यास समर्थन देण्यासाठी वापरतो 
```

Comment:
Javascript should be in English as it is

Reply:
Corrected. I request you to check. 
Thanks again!